frontend/test/components/measuretable.spec.jsx

```diff
@@ -2,6 +2,8 @@ import { shallow, configure } from 'enzyme';
 import Adapter from 'enzyme-adapter-react-16';
 import MeasureTable from '../../ui/measuretable';
 
+Date.now = jest.fn(() => 1482363367071);
```

Comment:
Let's make this a little more human readable:

```js
const referenceDate = new Date("2018-11-20T15:00:00Z");
Date.now = jest.fn(() => referenceDate.getTime());
```

I think then you can revert the changes below to change lastUpdated to use referenceDate instead of Date.now(), and the snapshots should be more realistic (instead of swapping a reference date for an integer). Or at least I think so. Give it a try. :smile:

Reply:
> and the snapshots should be more realistic (instead of swapping a reference date for an integer)

I have updated the reference date. I am sorry I don't get what you mean by this? :sweat_smile:

Reply:
Oh, I just mean it will more realistically reflect the data that is passed into measureTable in the real world. There we pass a date string, not an integer representation of unix time.